src/factory.ts

```diff
@@ -45,13 +45,15 @@ import {GitHubChangelogNotes} from './changelog-notes/github';
 import {DefaultChangelogNotes} from './changelog-notes/default';
 import {BaseStrategyOptions} from './strategies/base';
 import {LinkedVersions} from './plugins/linked-versions';
+import {DotnetYoshi} from './strategies/dotnet-yoshi';
 
 // Factory shared by GitHub Action and CLI for creating Release PRs
 // and GitHub Releases:
 // add any new releasers you create to this type as well as the `releasers`
 // object below.
 const allReleaseTypes = [
   'dart',
+  'dotnet-yoshi',
```

Comment:
A good future refactor (_you've brought it up too_) would be making it so you only need to define a new strategy in one place.